front/pages/api/w/[wId]/assistant/conversations/[cId]/messages/index.ts

```diff
@@ -22,27 +22,8 @@ async function handler(
   >,
   auth: Authenticator
 ): Promise<void> {
-  const user = auth.user();
-  if (!user) {
-    return apiError(req, res, {
-      status_code: 404,
-      api_error: {
-        type: "workspace_user_not_found",
-        message: "Could not find the user of the current session.",
-      },
-    });
-  }
+  const user = auth.getNonNullableUser();
```

Comment:
this throws  rather than returning a "not found" which is a bit different; do we think it's fine?

Reply:
No this can't throw because the wrapper `allowUserOutsideCurrentWorkspace` checks for the non-nullity of owner and return a proper error otherwise :+1: